bin/httpd_configmap_generator

```diff
@@ -62,13 +63,21 @@ supported auth_type: #{HttpdConfigmapGenerator.supported_auth_types.join(', ')}
         end
       end
 
+      params[:host] = get_canonical_hostname(params[:host])
```

Comment:
I don't think params[:host] is required for an update or export type.  Maybe append that like with if params[:host] ?

Reply:
👍 Will do